src/bvers.f.in

```diff
@@ -0,0 +1,50 @@
+C> @file
+C> @authors J. Ator @date 2020-10-21
+	
+C> THIS SUBROUTINE RETURNS A CHARACTER STRING CONTAINING THE
+C>   VERSION NUMBER OF THE BUFR ARCHIVE LIBRARY SOFTWARE.
+C>
+C> PROGRAM HISTORY LOG:
+C> @li 2009-03-23  J. ATOR    -- ORIGINAL AUTHOR
+C> @li 2011-09-26  J. ATOR    -- UPDATED TO VERSION 10.0.1
+C> @li 2012-02-24  J. ATOR    -- UPDATED TO VERSION 10.1.0
+C> @li 2012-10-12  J. ATOR    -- UPDATED TO VERSION 10.2.0
+C> @li 2012-11-29  J. ATOR    -- UPDATED TO VERSION 10.2.1
+C> @li 2012-12-04  J. ATOR    -- UPDATED TO VERSION 10.2.2
+C> @li 2013-01-08  J. ATOR    -- UPDATED TO VERSION 10.2.3
+C> @li 2013-01-09  J. ATOR    -- UPDATED TO VERSION 10.2.4
+C> @li 2013-01-25  J. ATOR    -- UPDATED TO VERSION 10.2.5
+C> @li 2014-11-12  J. ATOR    -- UPDATED TO VERSION 11.0.0
+C> @li 2015-09-24  J. ATOR    -- UPDATED TO VERSION 11.0.1
+C> @li 2016-02-12  J. ATOR    -- UPDATED TO VERSION 11.0.2
+C> @li 2016-03-18  J. ATOR    -- UPDATED TO VERSION 11.1.0
+C> @li 2016-05-10  J. ATOR    -- UPDATED TO VERSION 11.2.0
+C> @li 2017-04-03  J. ATOR    -- UPDATED TO VERSION 11.3.0
+C> @li 2020-10-21  J. ATOR    -- UPDATED TO USE @PROJECT_VERSION@
+C>                           MACRO FROM CMAKE
+C>
+C> USAGE:    CALL BVERS (CVERSTR)
+C>
+C> @param[out]  OUTPUT ARGUMENT LIST:
+C>     CVERSTR  - CHARACTER*(*): VERSION STRING
+C>
+C> REMARKS:
+C> @li   THIS ROUTINE CALLS:        BORT
+C> @li   THIS ROUTINE IS CALLED BY: WRDLEN
```

Comment:
Some requested changes:
* lower-case the text in the documentation, so it does not seem like we are SHOUTING so much. ;-)
* take out the USAGE: line, doxygen already provides this based on looking at the code.
* clean up the spacing of the paragraph at the top of the documentaiton block. I converted to doxygen but did not clean up the spacing. Also note that the first sentence of the documentation has special meaning, it's called the "brief" description of the code. So in the case this will appear next to this function name in various places:

<pre>
THIS SUBROUTINE RETURNS A CHARACTER STRING CONTAINING THE VERSION NUMBER OF THE BUFR ARCHIVE LIBRARY SOFTWARE.
</pre>

* The "@param[out]" goes on the same line and just before the name of the parameter. Like this:
<pre>
C> @param[out] CVERSTR  - CHARACTER*(*): VERSION STRING
</pre>

* When referencing another function (like WRDLEN above) do it in lower-case and with parens: wrdlen() Then doxygen will automatically hyper-link to the function documentation.

* Change @li back to hypens. A hyphen is one char, @li is three. Please undo this change we would like all the doxygen documentation to have the same look and feel so that we can all edit any documentation. So whatever conventions we adopt, will be for *all* the doxygen documentation.

Reply:
OK, I will change the @li to hyphens, and I do also see that I had the @param[out] on the wrong line ;-)

The uppercase documentation is just something that was in the library historically (it's a very old library) and we kind of stuck with it.  But as I'm going through all of the subprograms now anyway to add doxygen markup, I'll go ahead and switch to lower-case as much as I can in the docblocks.  This is fairly straightforward in vi using the ~ key.